### PR TITLE
Add ERC: Multiplicative Tokens

### PR DESCRIPTION
**Pull request by Pandapip1**
Migration from https://github.com/ethereum/EIPs/pull/7873

### CODE REVIEW

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by sullof** [comment]
@Pandapip1 I wonder why this PR passes the automatic checks, while all the others have issue with files other than the files in the PR. Any suggestion?

File: ERCS/erc-7538.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Motivation
+
+Many projects necessitate the creation of various types of tokens, both fungible and non-fungible. While certain standards are ideal for this purpose, they lack support for fractional tokens. Additionally, some tokens may require built-in inflation or deflation mechanisms, or may wish to allow transfers in unconventional increments, such as `0.5`.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+The `MultiplierMetadata` interface MUST be implemented in the resolved ERC-1046 `tokenURI` of tokens that use a `multiplier`:
```
Comment (xinbenlv) [inline ERCS/erc-7538.md:26]:
```suggestion
The `MultiplierMetadata` interface MUST be implemented in the resolved [ERC-1046](./erc-1046.md) `tokenURI` of tokens that use a `multiplier`:
```

File: ERCS/erc-7538.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-10-18
+requires: 20, 1046, 1155
+---
+
+## Abstract
+
+This EIP extends [ERC-1046](./eip-1046.md)-compatible token types (notably, [ERC-20](./eip-20.md) and [ERC-1155](./eip-1155.md) by introducing a `multiplier` field to the metadata schema, altering how user-facing balances are displayed.
```
Comment (xinbenlv) [inline ERCS/erc-7538.md:16]:
```suggestion
This EIP extends [ERC-1046](./erc-1046.md)-compatible token types (notably, [ERC-20](./erc-20.md) and [ERC-1155](./erc-1155.md) by introducing a `multiplier` field to the metadata schema, altering how user-facing balances are displayed.
```

**Review by xinbenlv — APPROVED** [review]
Good enough as draft

**Pull request merged**